README.md

```diff
@@ -116,11 +113,8 @@ packages:
     - package: dbt-labs/dbt_utils
       version: [">=1.0.0", "<2.0.0"]
 
-    - package: calogica/dbt_expectations
-      version: [">=0.8.0", "<0.9.0"]
-
     - package: calogica/dbt_date
-      version: [">=0.7.0", "<0.8.0"]
+      version: [">=0.9.0", "<0.10.0"]
```

Comment:
dbt_date is a dependency of dbt_expectations. So we can remove this from the list since it is no longer being used.

Reply:
Whoops missed this, updated